kura/org.eclipse.kura.linux.bluetooth/src/main/java/org/eclipse/kura/linux/bluetooth/le/BluetoothGattImpl.java

```diff
@@ -276,6 +278,51 @@ public void processInputStream(String string) {
     public void processErrorStream(String string) {
     }
 
+    @Override
+    public BluetoothGattSecurityLevel getSecurityLevel() throws KuraException {
+
+        BluetoothGattSecurityLevel level = BluetoothGattSecurityLevel.UNKNOWN;
+        if (this.m_connected && this.m_proc != null) {
+            this.m_securityLevel = "";
```

Comment:
Uhm, I mean could the securityLevel be of type BluetoothGattSecurityLevel?
Also processLine maybe could return an object obtained from parsing. Maybe it's slightly more readable.